pkgs/development/python-modules/iterfzf/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, buildPythonPackage, fzf, fetchFromGitHub }:
+
+buildPythonPackage rec {
+  pname = "iterfzf";
+  version = "0.5.0.20.0";
+
+  src = fetchFromGitHub {
+    owner = "dahlia";
+    repo = pname;
+    rev = version;
+    sha256 = "0nrfq1zj4pwlgyd8ra5wl7skdbzvnxdh7wwhyh83y7yax8sp1xvj";
+  };
```

Comment:
I'd recommend using the `fetchPypi` function instead (this way you'll benefit from automatic updates):
```nix
  src = fetchPypi {
    inherit pname version;
    sha256 = "0by5pk7w1j2xzhj4l0nw09sz9025pdskbf1b7k87kfkwgbc08vrb";
  };
```